docs/reference/inference/put-inference.asciidoc

```diff
@@ -100,18 +103,25 @@ the same name and the updated API key.
 
 `embedding_type`::
 (Optional, string)
-Specifies the types of embeddings you want to get back. Defaults to `float`.
+Only for `text_embedding`. Specifies the types of embeddings you want to get
+back. Defaults to `float`.
 Valid values are:
   * `byte`: use it for signed int8 embeddings (this is a synonym of `int8`).
   * `float`: use it for the default float embeddings.
   * `int8`: use it for signed int8 embeddings.
 
 `model_id`::
 (Optional, string)
-The name of the model to use for the {infer} task. To review the available
-models, refer to the
-https://docs.cohere.com/reference/embed[Cohere docs]. Defaults to
-`embed-english-v2.0`.
+The name of the model to use for the {infer} task.
+Available values for `rerank`:
+* `rerank-english-v3.0`,
+* `rerank-multilingual-v3.0`,
+* `rerank-english-v2.0`,
+* `rerank-multilingual-v2.0`.
+
+To review the available `text_embedding` models, refer to the
+https://docs.cohere.com/reference/embed[Cohere docs]. The default value for
+`text_embedding` is `embed-english-v2.0`.
```

Comment:
I think rather than listing all of the available models, it probably makes more sense to just mention that the available models are listed on the cohere website. Then we won't need to update our docs when they release new models.

Reply:
https://docs.cohere.com/reference/rerank-1